dataservice_publisher/app.py

```diff
@@ -44,6 +51,93 @@ async def authenticated(request: web.Request) -> bool:
     return False
 
 
+async def prepare_mime_types(accept_mime_types: List[str]) -> List[str]:
+    """Prepare the accept mime types and sort on q-parameter."""
+    logging.debug(f"Prcoessing accept mime types: {accept_mime_types}")
+    # Assign q-parameter:
+    accept_mime_types_sorted: List[Dict[str, str]] = []
+    for mime_type in accept_mime_types:
+        mime_type_split = mime_type.split(";")
+        mime_type_with_q = {
+            "type": mime_type_split[0],
+            "q": "1",
+        }
+        if len(mime_type_split) == 1:
+            accept_mime_types_sorted.append(mime_type_with_q)
+        else:
+            for mime_type_part in mime_type_split:
```

Comment:
Her er det vel unødvendig (feil) å behandle første element, fordi første er typen og ikke et _key_ _value_ pair.
```suggestion
            for mime_type_part in mime_type_split[1:]:
```

Reply:
Fiksa i 36f5295